addons/viewport/.storybook/addons.js

```diff
@@ -0,0 +1 @@
+import '@storybook/addon-viewport/register';
```

Comment:
Please remove the dev setup for this addon, and use the cra-kitchen-sink to develop on.